src/Calender.hpp

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include <unordered_map>
+#include "Lesson.hpp"
+#include "User.hpp"
+
+
+
+class Calender
```

Comment:
shouldn't it be a Calendar instead of Calender (typo)?

Reply:
Fixed